stories/file-uploader/file-uploader.stories.js

```diff
@@ -0,0 +1,80 @@
+import '../../dist/button.css';
+import '../../dist/icon.css';
+import '../../dist/form-item.css';
+import '../../dist/file-uploader.css';
+import '../../dist/form-label.css';
+import '../../dist/input.css';
+
+export default {
+    title: 'Components/FileUploader',
+    parameters: {
+        description: `
+The file uploader control allows the user to select the files from the system to upload.
```

Comment:
can you rename `control` to `component`?

Reply:
Modified the documents and pushed.